src/tests/ftest/util/test_utils_pool.py

```diff
@@ -95,6 +96,7 @@ def create(self):
             "uid": self.uid,
             "gid": self.gid,
             "scm_size": self.scm_size.value,
+            "label": self.label
```

Comment:
Use a new `label_generator` object to provide a unique pool label whenever its `get_label()` method is called.  If the current pool label is None, just return None:
```suggestion
            "label": self.label_generator.get_label(self.label.value)
```